searx/engines/bing_news.py

```diff
@@ -134,17 +124,20 @@ def response(resp):
 
 
 def fetch_traits(engine_traits: EngineTraits):
-    """Fetch languages and regions from Bing-News.
+    """Fetch languages and regions from Bing-News."""
+    # pylint: disable=import-outside-toplevel
+
+    from searx.engines.bing import fetch_traits as _f
 
-    The :py:obj:`description <searx.engines.bing_news.bing_traits_url>` of the
-    first table says *"query parameter when calling the Video Search API."*
-    .. that's why I use the 4. table "News Category API markets" for the
-    ``xpath_market_codes``.
+    _f(engine_traits)
 
-    """
+    # fix market codes not known by bing news:
 
-    xpath_market_codes = '//table[4]/tbody/tr/td[3]'
-    # xpath_country_codes = '//table[2]/tbody/tr/td[2]'
-    xpath_language_codes = '//table[3]/tbody/tr/td[2]'
+    # In bing the market code 'zh-cn' exists, but there is no 'news' category in
+    # bing for this market.  Alternatively we use the the market code from Honk
+    # Kong.  Even if this is not correct, it is better than having no hits at
+    # all, or sending false queries to bing that could raise the suspicion of a
+    # bot.
 
-    _fetch_traits(engine_traits, bing_traits_url, xpath_language_codes, xpath_market_codes)
+    # HINT: 'en-hk' is the region code it does not indicate the language en!!
+    engine_traits.regions['zh-CN'] = 'en-hk'
```

Comment:
@BernieHuang2008 this will alias the bing-news for `:zh` and `:zh-CN` to `:zh-HK` .. not sure if it is a good or bead idea of mine / may you know a better alternative?

Reply:
btw, what do you mean "no `:zh-cn`"?
![image](https://github.com/searxng/searxng/assets/88757735/538c3f5b-a04b-47e8-977e-295300431141)
if im correct, it's obvious that the very second result are different.

Reply:
I SearXNG we have a [Search Syntax](https://docs.searxng.org/user/search-syntax.html) / with this syntax you can for instance select a language/region .. as you can do it in the drop-down menu in the left upper:
 
![image](https://github.com/searxng/searxng/assets/554536/2310705a-7b23-46ce-80e1-31b94a238e55)

When write down a search term here, I use this syntax / for instance `:zh-TW !bing bmw` is a search term that will use the bing engine with language Chinese in region Taiwan to query the word "bmw".

Note: You cant use this syntax on bing.com as you have done above in your screenshot.